de-DE/Common.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  'Navigation': {
+    'Sessions': 'Sessions',
+    'Server': 'Server Einstellungen',
+    'Players': 'Spieler + Team',
+    'Badges': 'Orden',
+    'Pokédex': 'Pokédex',
+    'Web': 'Web Sources',
+    'Nuzlocke': 'Nuzlocke Tools',
+    'Settings': 'Einstellungen',
+    'About': 'About'
```

Comment:
Are the following correct German translations?

- Sessions
- Web Sources
- "Tools"
- About
-